OpenOversight/app/templates/list_officer.html

```diff
@@ -128,6 +116,18 @@ <h3 class="panel-title accordion-toggle">Age range</h3>
             </div>
           </div>
         </div>
+        <div class="panel">
+          <div class="panel-heading{% if not form_data['unique_internal_identifier'] %} collapsed{% endif %}" data-toggle="collapse" data-target="#filter-unique_internal_identifier">
+            <h3 class="panel-title accordion-toggle">Unique ID</h3>
+          </div>
+          <div class="collapse{% if form_data['unique_internal_identifier'] %} in{% endif %}" id="filter-unique_internal_identifier">
```

Comment:
Would the new filter work here too?

```suggestion
          <div class="collapse {{ form_data | field_in_query("unique_internal_identifier") }}" id="filter-unique_internal_identifier">
```

Reply:
Oh whoops, good catch